construbot/proyectos/views.py

```diff
@@ -542,6 +542,25 @@ def get_success_url(self):
             kwargs={'pk': self.kwargs['pk']}
         )
 
+    def get_context_data(self, **kwargs):
+        context = super(CatalogosView, self).get_context_data(**kwargs)
+        context['type'] = self.tipo
```

Comment:
```suggestion
        context['type'] = self.tipo if hasattr('tipo', self) else None
```
Checa si no truena si `self` no tiene el atributo `tipo`
Tambien lo que puedes hacer es darle el atributo `self.tipo` el valor de `None` y así ya no truena